src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  padding: $calendar-padding;
+  gap: $gap-size;
+
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    height: $day-size;
+    width: $day-size;
+    border: $border-size solid black;
+    background-color: #eee;
+    transition: all 0.5s;
+    }
+
+    &__day:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+
+    }
+
+    &__day::before {
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+    }
+  }
+
+    @for $day from 1 through 31 {
+      .calendar__day:nth-child(#{$day}) {
+        &::before {
+          content:'#{$day}';
+          }
+      }
+    }
+
+    @for $day from 28 through 31 {
```

Comment:
check this behavior. should change amount of days
<img width="1529" alt="Screenshot 2023-11-23 at 20 56 18" src="https://github.com/mate-academy/layout_calendar/assets/117587463/025c1af4-1a87-464c-88e0-39554900b2b7">